### ISSUE TITLE
Blacklist is downright confusing

### ISSUE THREAD

**Issue by 161134050** [Sycnex/Windows10Debloater#140]
In the GUI theres a button that says "Customize blacklist", it opens a window called "Customize whitelist and blacklist"

So what am I doing here? Is it a blacklist or a whitelist? Will checked boxes be added to a blacklist and therefore be removed? Or will checked boxes add to the whitelist?

Does the button called "Remove bloatware with blacklist" remove all bloatware on the blacklist? Or will it keep the software that is on the blacklist (for being removed) and remove everything else?

**Comment by Schytheron** [comment]
Yeah I am wondering this too... would have been way easier to understand if there was just a blacklist. I don't understand why the whitelist needs to exist...

**Comment by Sycnex** [comment]
I'm actually not certain but it seemed to me when I reviewed the code that it acted like a blacklist, not a whitelist.

**Comment by FireCulex** [comment]
I'm trying to figure this out myself. I wish someone would do a youtube vid.

**Comment by Dabbrivia** [comment]
@Sycnex so what will happen with the entries where the check boxes are filled, will they get removed?
